cmd/earth/main.go

```diff
@@ -794,6 +802,8 @@ func (app *earthApp) before(context *cli.Context) error {
 
 	app.buildkitdSettings.DebuggerPort = app.cfg.Global.DebuggerPort
 	app.buildkitdSettings.RunDir = app.cfg.Global.RunPath
+	app.buildkitdSettings.AdditionalArgs = app.buildkitdAdditionalArgs.Value()
```

Comment:
We should put this in the `config.yml` because any change to it requires buildkit restarts.

Reply:
What about the other args we pass through to buildkitd? It seemed 50-50 so I just went for one.

Reply:
I think all that's left in flag form is either deprecated settings or internal ones.

Config is better especially for CI environments like Jenkins where we want to avoid different pipelines from causing restarts.

Reply:
I see. Th other ones are passed in via different route; wont cause a restart since its not in the `docker run`. I'll move this then. Do we do any clever config merging from file -> env?

Reply:
Actually whatever is in `buildkitSettings` will cause a restart - we compare a hash of its JSON. Not sure what you mean by `file -> env`? Don't think we need env setting in this case.

Reply:
MIve used a lot of config packages that build a config starting from a file, then layering env vars, then CLI args. Looks like we dont. I have already made the changes.

Reply:
We have two remaining args that come in via the CLI - `GIT_URL_INSTEAD_OF` and `EARTHLY_BUILDKIT_CACHE_SIZE_MB`. Those should be moved out sometime. Probably another PR another time.

Reply:
:+1:

I believe both can be removed with @alexcb's latest ssh-related changes.

Reply:
`GIT_URL_INSTEAD_OF` is schedule to be removed, it no longer has an effect.